compiler/cpp/src/generate/t_erl_generator.cc

```diff
@@ -55,6 +55,7 @@ class t_erl_generator : public t_generator {
     out_dir_base_ = "gen-erl";
 
     legacy_names_ = (parsed_options.find("legacynames") != parsed_options.end());
+    use_maps_ = (parsed_options.find("usemaps") != parsed_options.end());
```

Comment:
I think `maps` would be a more consistent option name.  Similar existing options are `slots` (for Python) and `templates` (for C++), which don't include a **use** prefix.